README.md

```diff
@@ -8,7 +8,7 @@
         <a href="https://github.com/process-analytics/bpmn-visualization-js/releases">
           <img alt="GitHub release (latest by date including pre-releases)" src="https://img.shields.io/github/v/release/process-analytics/bpmn-visualization-js?color=orange&include_prereleases"> 
         </a> 
-        <a href="https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/demo/index.html">
+        <a href="https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/index.html">
```

Comment:
:+1:  for the badge update
:question: shouldn't we change the screenshot as Hacktoberfest is done?

Reply:
Yes, I asked me the same question the last week 🙂

Reply:
yes, definitely